core/src/test/java/nl/nn/adapterframework/util/FilenameUtilsTest.java

```diff
@@ -0,0 +1,272 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+/**
+ * FilenameUtils Tester.
+ *
+ * @author <Sina Sen>
+ */
+public class FilenameUtilsTest {
```

Comment:
This class didn't require any tests. It's a utilities class copied from apache commons io.
The only reason it's here is in order to avoid websphere compilations and runtime issues.

Reply:
okay, more communication will be assured on testing new classes.